### PR TITLE
hardening/1369_fiware_license_headers

### PR DESCRIPTION
**Pull request by frbattid**
* Implements issue #1369

### CODE REVIEW

File: cygnus-common/README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Simply configure the FIWARE repository if not yet configured:
     $ cat > /etc/yum.repos.d/fiware.repo <<EOL
     [Fiware]
     name=FIWARE repository
-    baseurl=http://repositories.testbed.fi-ware.eu/repo/rpm/x86_64/
+    baseurl=http://repositories.testbed.fiware.eu/repo/rpm/x86_64/
```
Comment (fgalan) [inline cygnus-common/README.md]: Should be .org

File: cygnus-ngsi/README.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ Simply configure the FIWARE repository if not yet configured:
     $ cat > /etc/yum.repos.d/fiware.repo <<EOL
     [Fiware]
     name=FIWARE repository
-    baseurl=http://repositories.testbed.fi-ware.eu/repo/rpm/x86_64/
+    baseurl=http://repositories.testbed.fiware.eu/repo/rpm/x86_64/
```
Comment (fgalan) [inline cygnus-ngsi/README.md]: Should be .org

Comment (fgalan) [inline cygnus-ngsi/README.md]: Should be .org

File: cygnus-common/README.md (hunk 1 above)
Comment (frbattid) [inline cygnus-common/README.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1375/commits/b0b2e859498b6c1a20112df2a0b373f9b41dade4

File: cygnus-ngsi/README.md (hunk 2 above)
Comment (frbattid) [inline cygnus-ngsi/README.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1375/commits/b0b2e859498b6c1a20112df2a0b373f9b41dade4

Comment (frbattid) [inline cygnus-ngsi/README.md]: This is the same than above. Anyway, fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1375/commits/b0b2e859498b6c1a20112df2a0b373f9b41dade4

**Comment by fgalan** [comment]
LGTM

**Pull request merged**